validate/BUILD

```diff
@@ -55,7 +55,7 @@ java_proto_library(
 
 alias(
     name = "go_default_library",
-    actual = ":validate_go",
```

Comment:
I don't understand. If we've removed this, why are we changing to `//validate:validate_go` elsewhere in this diff?

Reply:
adding offline comments here....

The only thing i did is get the dockerfile to run on arm machines and update the go version, consequently the go mod and bazel/repositories.bzl with later versions…once i did this, i synced up the bazel repos from the go mod with
```bash
bazel run //:gazelle -- update-repos -from_file=go.mod -prune -to_macro=dependencies.bzl%go_third_party
```
thats when it went as made the huge diff :stuck_out_tongue: I dont have a bazel-vocab so please excuse me. You can see everything bazel/gazelle did in commit [ea5d03e](https://github.com/envoyproxy/protoc-gen-validate/pull/629/commits/ea5d03ecf78f12c1c424e267c196b4da108e4f3b)… Updating bazel rules in `bazel/repositories.bzl` was all me - needed to happen to enable an arm build. Trying to simulate amd was impossible, my machine quickly ran out of memory.